<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DeleteWindow</class>
 <widget class="QDialog" name="DeleteWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>386</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>386</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>386</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>delete_window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">border:none;</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::Shape::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Shadow::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QFrame" name="frame_9">
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_27">
         <item>
          <widget class="QFrame" name="frame_15">
           <property name="styleSheet">
            <string notr="true">border:None</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::Shape::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Shadow::Raised</enum>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_18">
            <item>
             <widget class="QFrame" name="frame_16">
              <property name="styleSheet">
               <string notr="true">border:None;</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::Shape::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Shadow::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_19">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_30">
                 <property name="styleSheet">
                  <string notr="true">border:None;</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_20">
                  <item>
                   <widget class="QLabel" name="label_fio_user">
                    <property name="text">
                     <string>ФИО пациента (Архипенко Михаил Иванович)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_32">
                    <property name="frameShape">
                     <enum>QFrame::Shape::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_16">
                     <property name="spacing">
                      <number>4</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="line_edit_user">
                       <property name="accessibleDescription">
                        <string/>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid rgb(197, 197, 197);
border-radius:6px;
padding:4px;</string>
                       </property>
                       <property name="placeholderText">
                        <string>Введите ваше ФИО</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="button_user">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../qrc/main_icons.qrc">
                         <normaloff>:/icons/icons/user2.png</normaloff>:/icons/icons/user2.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>27</width>
                         <height>27</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_33">
                 <property name="styleSheet">
                  <string notr="true">border:None;</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_21">
                  <item>
                   <widget class="QLabel" name="label__address">
                    <property name="text">
                     <string>Адрес прописки (г. Минск, ул. Лобанка 23, кв. 55)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_34">
                    <property name="frameShape">
                     <enum>QFrame::Shape::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_17">
                     <property name="spacing">
                      <number>4</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="line_edit_address">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid rgb(197, 197, 197);
border-radius:6px;
padding:4px;</string>
                       </property>
                       <property name="placeholderText">
                        <string>Введите адрес прописки</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="button_address">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../qrc/main_icons.qrc">
                         <normaloff>:/icons/icons/address.png</normaloff>:/icons/icons/address.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>27</width>
                         <height>27</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_35">
                 <property name="styleSheet">
                  <string notr="true">border:None;</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_23">
                  <item>
                   <widget class="QLabel" name="label__date_of_birthday">
                    <property name="text">
                     <string>Дата рождения (20.02.2006)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_36">
                    <property name="frameShape">
                     <enum>QFrame::Shape::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_19">
                     <property name="spacing">
                      <number>4</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="line_edit_date_of_birthday">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid rgb(197, 197, 197);
border-radius:6px;
padding:4px;</string>
                       </property>
                       <property name="placeholderText">
                        <string>Введите дату рождения</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="button_of_birthday">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../qrc/main_icons.qrc">
                         <normaloff>:/icons/icons/date.png</normaloff>:/icons/icons/date.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>27</width>
                         <height>27</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_37">
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_24">
                  <item>
                   <widget class="QLabel" name="label_date_of_admission">
                    <property name="accessibleName">
                     <string/>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">border:None;</string>
                    </property>
                    <property name="text">
                     <string>Дата приема (20.02.2006)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_38">
                    <property name="frameShape">
                     <enum>QFrame::Shape::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_20">
                     <property name="spacing">
                      <number>4</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="topMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="line_edit_date_of_admission">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid rgb(197, 197, 197);
border-radius:6px;
padding:4px;</string>
                       </property>
                       <property name="placeholderText">
                        <string>Введите дату приема</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="button_of_date_of_addmission">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../qrc/main_icons.qrc">
                         <normaloff>:/icons/icons/date.png</normaloff>:/icons/icons/date.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>27</width>
                         <height>27</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QFrame" name="frame_39">
                 <property name="styleSheet">
                  <string notr="true">border:None;</string>
                 </property>
                 <property name="frameShape">
                  <enum>QFrame::Shape::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Shadow::Raised</enum>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_25">
                  <item>
                   <widget class="QLabel" name="label_of_fio_doctor">
                    <property name="text">
                     <string>ФИО врача (Архипенко Михаил Иванович)</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QFrame" name="frame_40">
                    <property name="frameShape">
                     <enum>QFrame::Shape::StyledPanel</enum>
                    </property>
                    <property name="frameShadow">
                     <enum>QFrame::Shadow::Raised</enum>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_21">
                     <property name="spacing">
                      <number>4</number>
                     </property>
                     <property name="leftMargin">
                      <number>0</number>
                     </property>
                     <property name="rightMargin">
                      <number>0</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QLineEdit" name="line_edit_fio_doctor">
                       <property name="styleSheet">
                        <string notr="true">border: 1px solid rgb(197, 197, 197);
border-radius:6px;
padding:4px;</string>
                       </property>
                       <property name="placeholderText">
                        <string>Введите ФИО врача</string>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="button_of_doctor">
                       <property name="text">
                        <string/>
                       </property>
                       <property name="icon">
                        <iconset resource="../qrc/main_icons.qrc">
                         <normaloff>:/icons/icons/doctor2.png</normaloff>:/icons/icons/doctor2.png</iconset>
                       </property>
                       <property name="iconSize">
                        <size>
                         <width>27</width>
                         <height>27</height>
                        </size>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="frame_3">
        <property name="styleSheet">
         <string notr="true">border:None
</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::Shape::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Shadow::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <property name="spacing">
          <number>10</number>
         </property>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>5</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_cancel">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(87, 99, 177);
border:1px solid rgb(87, 99, 177);;
border-radius:6px;
padding:2px 4px;</string>
           </property>
           <property name="text">
            <string>Отмена</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_find">
           <property name="styleSheet">
            <string notr="true">background-color:rgb(67, 153, 143);
border:1px solid rgb(67, 153, 143);
border-radius:6px;
padding:2px 4px;</string>
           </property>
           <property name="text">
            <string>Найти</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Orientation::Horizontal</enum>
           </property>
           <property name="sizeType">
            <enum>QSizePolicy::Policy::Fixed</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>15</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../qrc/main_icons.qrc"/>
 </resources>
 <connections/>
</ui>
